configs/ib_config.yaml

```diff
@@ -1,8 +1,6 @@
---- 
-generate:
-  - client
-  - types
+---
 package: image_builder
+generate:
+  models: true
```

Comment:
this did not work for me adding the models here, it did not recognize this option. and it is not written as an option in oapi.
Can you explain?

Reply:
I think they've changed the options in the new version, but I can't find documentation for this new format...

Reply:
You used an old version, the current (1.11) only supports this syntax.

Go ahead and try my patch, checkout and then:

```
make install-tools
rm internal/clients/*/*.gen.go
make generate-clients
```

Reply:
Just to explain a little bit more, the key command you need to run is:

    go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@latest

This will get you 1.11 today.